docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -0,0 +1,1673 @@
+
+- `help`_
+- `background`_
+- `count`_
+- `cron`_
+- `delnode`_
+- `dmon`_
+- `edges`_
+- `feed`_
+- `graph`_
+- `iden`_
+- `layer`_
+- `lift`_
+- `limit`_
+- `macro`_
+- `max`_
+- `merge`_
+- `min`_
+- `model`_
+- `movetag`_
+- `parallel`_
+- `pkg`_
+- `ps`_
+- `queue`_
+- `reindex`_
+- `scrape`_
+- `service`_
+- `sleep`_
+- `spin`_
+- `splice`_
+- `tag`_
+- `tee`_
+- `tree`_
+- `trigger`_
+- `uniq`_
+- `view`_
+
+See :ref:`storm-ref-syntax` for an explanation of the syntax format used below.
+
+The Storm query language is covered in detail starting with the :ref:`storm-ref-intro` section of the Synapse User Guide.
+
+.. NOTE::
+
+  Storm commands, including custom commands, are added to the Cortex as **runtime nodes** ("runt nodes" - see :ref:`gloss-node-runt`) of the form ``syn:cmd``. These runt nodes can be lifted and filtered just like standard nodes in a Cortex.
+
+**Example**
+
+Lift the ``syn:cmd`` node for the Storm ``movetag`` command:
+
+
+.. storm-cli:: syn:cmd=movetag
+
+.. NOTE::
+
+  While the documentation for built-in Storm commands is relatively basic, the ``syn:cmd`` form within the data model includes additional secondary properties that can be used to make commands (particularly custom commands) more "self-documenting" to users, because detail about the command can be introspected directly within the data model from within Storm. For example, the ``:input`` and ``:output`` properties can be used to specify a list (:ref:`type-array`) of forms that can be input (submitted) to the command and a list of forms that may be created by the command, respectively.
+
+
+.. _storm-help:
+
+help
+----
+
+The ``help`` command (``storm help``) displays the list of available built-in commands and a brief message describing each command. Help on individual commands is available via ``<command> --help``.
+
+**Syntax:**
+
+
+.. storm-cli:: help --help
+
+.. _storm-background:
+
+background
+----------
+
+The ``background`` command allows you to execute a Storm query as a background task (e.g., to free up the cmdr CLI / Storm runtime for additional queries). Use of ``background`` is a "fire-and-forget" process - any status messages (warnings or errors) are not returned to the console, and if the query is interrupted for any reason, it will not resume.
+
+See also :ref:`storm-parallel`.
+
+**Syntax:**
+
+
+.. storm-cli:: background --help
+   
+.. _storm-count:
+
+count
+-----
+
+The ``count`` command enumerates the number of nodes returned from a given Storm query and displays the resultant nodes and associated node count.
+
+**Syntax:**
+
+
+.. storm-cli:: count --help
+    
+
+**Examples:**
+
+- Count the number of email address nodes:
+
+
+.. storm-pre:: inet:email | count
```

Comment:
This could be a real example of showing count. use pre to make two nodes and storm-cli to count them.

Reply:
updated